kt/api-generator/src/main/kotlin/godot/codegen/constants/Classes.kt

```diff
@@ -136,6 +136,20 @@ object GodotTypes {
     )
 }
 
+object GodotMeta {
+    object Int {
+        const val uint32 = "uint32"
```

Comment:
Same as Float, but I don't think that adding the unsigned integers is a good idea. Now we have many part of the Godot api that went from being regular integer to unsigned which will force many casting again.
If look at the usages of unsigned integers in the API, you'll see that it's not because they need this extra bit of data to get bigger values. It just means it can't be negative but in practice it has little value.
There is no benefit in forcing people to use an unsigned long for flags when connecting to a signal for example.